src/entities/Role.ts

```diff
@@ -0,0 +1,29 @@
+import { Entity, Column, BaseEntity, PrimaryGeneratedColumn, OneToMany} from "typeorm";
+import { Client } from "./Client";
+
+export enum RoleTypes{
+    ADMIN = 'admin',
+    USER = 'user',
+    TEST = 'test'
+}
+
+
+@Entity('role')
+export class Role extends BaseEntity {
+    @PrimaryGeneratedColumn()
+    role_id!: Number
```

Comment:
u dont need to define and id for each entity typeorm can do that for u, google it and let me know if u need any help

Reply:
Perfect, Thank youu